.pipelines/e2e-windows.yaml

```diff
@@ -1,6 +1,10 @@
 name: $(Date:yyyyMMdd)$(Rev:.r)_e2e
 
 pr:
+  branches:
+    include:
+    - main
+    - 'official/*'
   paths:
     include:
     - '.pipelines/*'
```

Comment:
We need to include `parts/windows` if `paths` is used to trigger the e2e when detecting the changes in these paths.